md/dcore_intf.md

```diff
@@ -55,3 +55,10 @@
 | en_pfd_cal_ext_ave         |         |                        |                  | Test         | out      |   0       |
 | en_int_dump_start          |         |                        |                  | Test         | out      |   0       |
 | int_dump_start             |         |                        |                  | Test         | out      |   0       |
+| tx_en_ext_max_sel_mux      |         |                        |                  | Test         | out      | 'b0       |
```

Comment:
These new registers mimic the functionality of the PI control registers for the analog core.  The main difference is that there is no MM-CDR for the TX.